pkg/cli/mirror/helm.go

```diff
@@ -62,7 +62,7 @@ func (h *HelmOptions) PullCharts(ctx context.Context, cfg v1alpha2.ImageSetConfi
 	c := downloader.ChartDownloader{
 		Out:     os.Stdout,
 		Keyring: "",
-		Verify:  downloader.VerifyNever,
+		Verify:  downloader.VerifyIfPossible,
```

Comment:
Would like to know the reason behind verification mode change here.

Reply:
@dinhxuanvu Thanks for catching this. I meant to separate this out as it does not have to do with the feature. The motivation behind the mode change is to allow signed Helm chart to be verified. I will remove and put in a new PR.